update system-properties.adoc documentation for historyWidget.descriptionLimit (#3652)
jenkins-infra/jenkins.io: content/doc/book/managing/system-properties.adoc

  description: |
    How many builds to show in the build history side panel widget.

- name: historyWidget.descriptionLimit
  tags:
  - feature
  - UI
  def: |
    `100`
  since: 2.223
  description: |

>     Defines a limit for the characters shown in the description field for each build row at the Build History column.
>     A positive Integer value (e.g. `300`) will define the limit, after that limit is reached (...) will be shown.
>     Value `-1` is for unlimited characters (disable). And `0` means No Description is shown.

Reviewer: ```suggestion
    Defines a limit for the characters shown in the description field for each build row in the Build History column.
    A positive integer (e.g. `300`) will define the limit.
    After the limit is reached (...) will be shown.
    The value `-1` disables the limit and allows unlimited characters in the build description.
    The value `0` shows no description.
```
